### PR TITLE
Implement new palette

### PR DESCRIPTION
**Pull request by gtenaschuk**

### CODE REVIEW

File: assets/tailwind.config.js [hunk 1]
```diff
@@ -1,12 +1,40 @@
...
-    "./js/**/*.js",
-    "./css/**/*.css",
-    "../lib/**/*.*ex",
     "../**/*.*exs",
-    "../../../config/*.*exs",
-
-    // We need to include the Palette dependency so the classes get picked up by JIT.
-    "../deps/palette/**/*.*ex"
-  ]
+    "./js/**/*.js",
```
Comment (mustela) [inline assets/tailwind.config.js:5]: @gtenaschuk if we remove `"./css/**/*.css",` we won't be able to add our own tailwind classes right?

**Review by mustela — APPROVED** [review]
LGTP I just left you one small comment.

File: assets/tailwind.config.js (hunk 1 above)
Comment (gtenaschuk) [reply assets/tailwind.config.js:5]: To do that you need to include your css classes or files on the app.css file, if you want to make your new custom css priority  it should go after palette import, if you want to make palette priority your custom css should go before palette.

**Comment by gtenaschuk** [comment]
@mustela the elixir version is failing, i will use this to implement the lastest version. Is that ok?

File: assets/tailwind.config.js (hunk 1 above)
Comment (gtenaschuk) [reply assets/tailwind.config.js:5]: Also, if you are talking about using tailwind core css that does not exist in palette, you just need to use them in your app components or in your templates, tailwind will add them to the final priv css when the deploy command is executed.

File: mix.exs [hunk 2]
```diff
@@ -47,6 +47,8 @@ defmodule LiveSup.MixProject do
     ]
   end
 
+  defp elixirc_paths(:dev), do: ["lib", "deps/palette/lib"]
```
Comment (mustela) [inline mix.exs:50]: I think we need to change this one 

```
@palette_path "deps/palette" # this one should be used in the other references


defp elixirc_paths(:dev), do: ["lib"] ++ elixirc_palette_path(File.exists?(@palette_path))
  defp elixirc_paths(_), do: ["lib"]

  defp elixirc_palette_path(true), do: ["deps/palette/lib"]
  defp elixirc_palette_path(false), do: []
```

**Review by mustela — CHANGES_REQUESTED** [review]
I just realised that we should make it optional to use palette locally, otherwise everyone should download palette with livesup.

**Comment by mustela** [comment]
> @mustela the elixir version is failing, i will use this to implement the lastest version. Is that ok?

Perfect!

**Review by mustela — APPROVED** [review]

**Pull request merged**